### ISSUE TITLE
Make Clans actually useful

### ISSUE THREAD

**Issue by ColorfulMarker** [MegaSa1nt/GMDprivateServer#285]
Clans are currently pretty useless, how about making them more useful?

Ideas:
- Add a Clan Chat
- Clan Rewards 
  - The Clan Creator can choose a path (similar to the ones in game, but with stats like stars, moons, etc.)
  - When a Clan gathers enough of that stat, they get a reward (picked by gdps hosts)

Will add any ideas that are also added to the [Discord Forum](https://discord.com/channels/1175853758509490176/1321976928223236218)

**Comment by tails1154** [comment]
yeah. this would be pretty good except the client would ave no idea how to get a reward from it i dont think. and building on this i think it would be neat to show like clan levels (maybe replace like the magic tab because who uses that) that only clan members can see

**Comment by MegaSa1nt** [comment]
We can use vault codes for clan rewards
Magic tab is level recommendations (for you page) in new core

**Comment by tails1154** [comment]
like idk replace uh...trending tab maybe

**Comment by MegaSa1nt** [comment]
Lmao
